### ISSUE TITLE
Cannot display AcquisitionMultipoint condition

### ISSUE THREAD

**Issue by 0b11001111** [pyhmsa/pyhmsa-gui#2]
Whenever I try to display a **AcquisitionMultipoint condition** in the Viewer the following error occurs.

```
Traceback (most recent call last):
  File "/home/lukas/Workspace/GFE/pyhmsa-viewer/pyhmsa_viewer/main.py", line 1023, in _on_condition_display_requested
    window = _ConditionSubWindow(uid, identifier, self.controller)
  File "/home/lukas/Workspace/GFE/pyhmsa-viewer/pyhmsa_viewer/main.py", line 871, in __init__
    widget.setParameter(condition)
  File "/home/lukas/Workspace/GFE/pyhmsa-gui/pyhmsa_gui/spec/condition/acquisition.py", line 146, in setParameter
    self._wdg_positions.setPositions(condition.positions)
  File "/home/lukas/Workspace/GFE/pyhmsa-gui/pyhmsa_gui/spec/condition/specimenposition.py", line 301, in setPositions
    self.setParameter(positions)
  File "/home/lukas/Workspace/GFE/pyhmsa-gui/pyhmsa_gui/spec/condition/specimenposition.py", line 295, in setParameter
    model.reset()
AttributeError: '_SpecimenPositionModel' object has no attribute 'reset'
```

Full log of the session:
[log.txt](https://github.com/pyhmsa/pyhmsa-gui/files/577964/log.txt)

**Comment by ppinard** [comment]
Should be fixed by commit 3b82bf6191a37ccb658ef8b0d6cd82814d4d3cbf

**Comment by 0b11001111** [comment]
It is. Thank you ;)
